mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -236,8 +235,7 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"region_name": {
 										Type:     schema.TypeString,
-										Optional: true,
-										Computed: true,
+										Required: true,
```

Comment:
[Q] Earlier this was `Optional` and now it is changed to `Required`. Will this break if clients do not pass `region_name`?

Reply:
Because I re-read again the docs and it seems it's required to use the region, so I changed to required, I can roll back to optional if you want. Since the region name it's using as the key name, if you don't put the name the key would be empty along with the field and values, even with optional you got an error saying that the region must be set if you don't set `region_name`

Reply:
This makes sense. I think we should keep it `Required` then.